packages/cli/src/lib/setup.ts

```diff
@@ -25,63 +25,63 @@ export class SetupBuilder {
         return `# Function to configure MCP servers for claude
 configure-mcp-servers() {
   # Ensure configuration file exists
-  if [ ! -f /home/agent/.claude.json ]; then
-    echo '{}' > /home/agent/.claude.json
-    chown agent:agent /home/agent/.claude.json
+  if [ ! -f $HOME/.claude.json ]; then
+    echo '{}' | sudo tee $HOME/.claude.json
   fi
 
-  jq '.mcpServers //= {}' /home/agent/.claude.json | \
+  jq '.mcpServers //= {}' $HOME/.claude.json | \
     jq '.mcpServers += { "package-manager": { "type": "http", "url": "http://127.0.0.1:8090/mcp" } }' \
-    > /tmp/agent-settings.json
-  mv /tmp/agent-settings.json /home/agent/.claude.json
+      | sudo tee /tmp/agent-settings.json
+  sudo mv /tmp/agent-settings.json $HOME/.claude.json
+  sudo chown $(id -u):$(id -g) $HOME/.claude.json
```

Comment:
Is `sudo` and `chown` required here? Since the container runs using the `--user` option, the current user should be already right. Just wondering, I'm still wrapping my mind about users and docker.